interfaces/builtin/optical_drive.go

```diff
@@ -30,8 +30,8 @@ const opticalDriveBaseDeclarationSlots = `
 
 const opticalDriveConnectedPlugAppArmor = `
 # Allow read access to optical drives
```

Comment:
Can you also adjust this comment for 'and write'?

Reply:
Change made as requested.